README.md

```diff
@@ -97,138 +97,148 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
-### File Locksmith
+### Crop and Lock
+ - Fixed a Crop and Lock crash that would occur when trying to reparent a window crashes the target application. An error message is shown instead.
 
- - Icon update. Thanks [@jmaraujouy](https://github.com/jmaraujouy)!
- 
-### File Explorer add-ons
+### FancyZones
 
- - Fixed issue causing thumbnail previewers to lock files.
- - Open URIs from developer files in default browser. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Set the process and main thread priority to normal.
+ - Fixed handling newly created windows on Windows 11.
+ - Fixed scenarios where opening the FancyZones Editor would reset the layouts.
 
-### Installer
+### File Explorer add-ons
 
- - Fixed PowerToys autorun after installing as SYSTEM user. 
- - Removed CreateScheduledTask custom action to handle task creation only from runner code.
+ - Optimized CPU usage for generating SVG thumbnails.
+ - Improved handling of Gcode Thumbnails, including JPG and QOI formats. Thanks [@pedrolamas](https://github.com/pedrolamas)!
+ - Better handled errors when sending telemetry, which were causing reported crashes.
+ - Fixed some thumbnails not being shown centered like before the optimization.
 
-### Image Resizer
+### File Locksmith
 
- -  Moved from ModernWPF to WpfUI to refresh and modernize UI/UX. Thanks [@niels9001](https://github.com/niels9001)!
+ - Shows files opened by processes with PID greater than 65535. Thanks [@poke30744](https://github.com/poke30744)!
+ - Fixed a GDI object leak in the context menu which would crash Explorer.
+ 
+### Find My Mouse
 
-### Keyboard Manager
+ - Added new activation methods, including by hotkey. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
- - Rephrased labels to enhance clarity. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
- - Keyboard manager now allows Numpad keys to be bound separately from the main keyboard. If using a previous configuration, only the main keyboard (not numpad) keys will be mapped. To restore the previous behavior, manually add separate bindings for numpad keys.
+### Hosts File Editor
 
-### Mouse Highlighter
+ - Ignore the default ACME sample entries in the hosts file. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Improved save error handling and added better error messages. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Corrected a check for an error when signaling the application to start as administrator.
+ - Refactored the context menu. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Fixed dialogs overlapping the title bar after the upgrade to Windows App SDK 1.4. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - The move up and move down buttons in the context menu are disabled when filtering. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
```

Comment:
```suggestion
```
Don't need to add this. The regression was caused by the context menu refactoring in this release cycle. Menu items are disabling as expected in 0.73.0.